cloud-config/deploy-run-scripts/conf-apache/the-rest.conf

```diff
@@ -123,39 +123,49 @@ RewriteRule ^(/ENCODE(/|/.+/))index$    $1
+RewriteRule ^/ENCODE/analysisTools$    /search/?type=Software&purpose=community\ resource    [last,redirect=permanent]
 RewriteRule ^/ENCODE/experiment_guidelines$    /about/experiment-guidelines/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/platform_characterization$    /data-standards/platform-characterization/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/qualityMetrics$    /data-standards/2012-quality-metrics/?    [last,redirect=permanent]
-RewriteRule ^/ENCODE/contributors$    /about/contributors/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/contributors.*$    /help/project-overview/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/analysis$    /about/2012-integrative-analysis/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/pubsOther$    /publications/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/pubsEncode$    /publications/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/fileFormats$    /help/file-formats/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/contacts$    /help/contacts/?    [last,redirect=permanent]
-RewriteRule ^/ENCODE/FAQ/$    /tutorials/?    [last,redirect=permanent]
-RewriteRule ^/ENCODE/usageResources$    /tutorials/?    [last,redirect=permanent]
-RewriteRule ^/ENCODE/releaseLog$    /about/contributors/?    [last,redirect=permanent]
-RewriteRule ^/ENCODE/pilot$    /about/contributors/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/FAQ/$    /help/FAQ/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/usageResources$    /help/events/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/releaseLog$    /help/project-overview/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/pilot$    /help/project-overview/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/downloads$    /help/getting-started/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/downloadsMouse$    /help/getting-started/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/otherTerms$    /help/getting-started/?    [last,redirect=permanent]
-RewriteRule ^/ENCODE/integrativeAnalysis/VM$    http://encodedcc.stanford.edu/ftp/encodevm/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/aboutScaleup$    /help/project-overview/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/newsarch$    /search/?type=Page&news=true&status=released/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/integrativeAnalysis/VM$    http://ftp.encodedcc.org/encodevm/?    [last,redirect=permanent]
 RewriteRule ^/ENCODE/dataStandards$    /data-standards/?    [last,redirect=permanent]
+RewriteRule ^/ENCODE/pubs$    /publications/?    [last,redirect=permanent]
 RewriteCond %{REQUEST_METHOD}    =GET
 RewriteRule ^/encyclopedia/visualize    http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&hgt.customText=http://bib.umassmed.edu/~iyers/encode_elements/display/tracks.txt    [last,redirect=permanent]
+RewriteRule ^/search/?type=experiment(.*$)    /search/?type=Experiment$1    [last,redirect=permanent]
```

Comment:
I expect to remove this line- it doesn't work, and isn't really in the scope of these other redirects